docs/configuration/http_conn_man/headers.rst

```diff
@@ -162,3 +162,15 @@ following features are available:
   <config_http_conn_man_runtime_random_sampling>` runtime setting or via forced tracing using the
   :ref:`config_http_conn_man_headers_x-envoy-force-trace` and
   :ref:`config_http_conn_man_headers_x-client-trace-id` headers.
+
+.. _config_http_conn_man_headers_x-ot-span-context:
+
+x-ot-span-context
```

Comment:
Is it worth documenting that the actual format of this is base64 "binary OT carrier?" This is quite LS specific currently. I guess I would probably mention that this is LS specific and when Zipkin support is added we can update the docs to cover the differences between the 2 providers.